app/views/layouts/base.html.erb

```diff
@@ -32,6 +32,9 @@
       <% end %>
     </script>
     <%= javascript_include_tag "locale/#{FastGettext.locale}/app" %>
+    <% for plugin in ::Foreman::Plugin.all do %>
+      <%= javascript_include_tag "locale/#{FastGettext.locale}/#{plugin.engine.class.to_s.underscore.split('/', 2).first}" %>
```

Comment:
Does this work on `foreman-tasks`? Perhaps because the domain is `foreman_tasks`, but just want to make sure.

Reply:
Are you starting to understand why I'd love to rename it? Even though it's a pain once, it'll be easier in the long run.